### PR TITLE
Automate jumpbox post install steps

### LINKED ISSUE #1119: Jumpbox Setup Has Too Many Manual Steps
https://microsoft.github.io/AzureTRE/tre-admins/setup-instructions/configuring-shared-services/

The instructions for setting up the jumpbox have a large number of manual steps.
These should be converted into scripted steps.
Ideally, many of these steps should be moved into deployment/setup automation.

At present, the steps are manual and there is little to no guidance on the setup (e.g. which settings to set while running Windows installers)

### PR DESCRIPTION
**Pull request by chboudry**
Closes #1119 

Automate jumbbox post install : 

- Download & install Git
- Download & install AzureCli
- Download JQ
- Clone Azure TRE in C:/AzureTRE

Removed documentation to do those steps manually.

### CODE REVIEW

File: docs/tre-admins/setup-instructions/configuring-shared-services.md [hunk 1]
```diff
@@ -2,16 +2,6 @@
...
-2. Download Git for Windows from [https://git-scm.com/download/win](https://git-scm.com/download/win) and install
-3. Download Azure CLI from [https://aka.ms/installazurecliwindows](https://aka.ms/installazurecliwindows) and install
-4. Open Git Bash
-5. Login to Azure ```az login``` and set the default subscription if needed: ```az account set --subscription <subscription_id>```
-6. Git clone the TRE repository: ```git clone https://github.com/microsoft/AzureTRE.git```
-7. Download jq ```curl -L -o /usr/bin/jq.exe https://github.com/stedolan/jq/releases/latest/download/jq-win64.exe```
-
 ## Configure Nexus repository
 
 1. Run the Nexus configuration script to reset the password and setup a PyPI proxy on Nexus:
```
Comment (marrobi) [inline docs/tre-admins/setup-instructions/configuring-shared-services.md:7]: Do we need to specify where to run the script from now the repo is cloned?

Comment (chboudry) [reply docs/tre-admins/setup-instructions/configuring-shared-services.md:7]: that's a good point I will update the doc

Comment (chboudry) [reply docs/tre-admins/setup-instructions/configuring-shared-services.md]: done

**Pull request merged**